andela-coop/andelacoop-backend
# AC-1 Create backend repo

#### What Does This PR Do?
It creates backend repo

#### Description Of Task To Be Completed
- Configure babel
- Add .env.sample for environment variable guide
- Setup database configuration
- Setup test
- Create a simple endpoint to get all users

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
- Pull the branch
- create a .env file (please check .env.sample for the necessary env variable)
- create a database and update the env. with necessary details
- Run `yarn install` and `yarn db:roll:migrate` to do migration
- Run `yarn db:reseed` to seed the user database
- Do a GET operation on `localhost:5000/api/users` to get the seeded user

#### What are the relevant stories?
[AC-1](https://andela-coop.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=AC&selectedIssue=AC-1)

#### Screenshot(s)
N/A
